lib/util/subtokenize.js

```diff
@@ -155,13 +155,21 @@ function unravelLinkedTokens(token, context) {
   if (hasGfmTaskListHack) {
     tokenizer._gfmTasklistFirstContentOfListItem = false
   }
+}
+
+function unravelLinkedTokens(token, context) {
+  while (token) {
+    // Loop over the tokens because a recursive function would cause a stackoverflow
+    unravelLinkedTokensImpl(token, context)
 
-  if (token.next) {
-    // Unravel the next token.
-    unravelLinkedTokens(token.next, context)
-  } else {
-    // Done!
-    divideTokens(token)
+    if (token.next) {
+      // Unravel the next token.
+      token = token.next
+    } else {
+      // Done!
+      divideTokens(token)
+      return
+    }
   }
```

Comment:
```suggestion
    // Unravel the next token.
    token = token.next
  }

  // Done!
  divideTokens(token)
}
```

This should work I believe?

Reply:
It seems to be broken since token would be undefined but we still need to pass it to `divideTokens`